setup.cfg

```diff
@@ -0,0 +1,3 @@
+[mypy]
+files=schedule
```

Comment:
Why do we need this?

Reply:
By adding this, we can run `mypy` instead of `mypy schedule` or `mypy .`. When we run `mypy .`, it also checks the `docs`. For testing, I tried `mypy --strict`. I was tempted to also touch the docs with probably is not what we want - hence this :-)

Reply:
Aaah that's pretty cool! Can we try to add this configuration to the `setup.cfg` file to keep configuration a bit more centralized?

Reply:
I'm confused now. Which configuration do you want to add to the `setup.cfg`?

Reply:
Whauw, I totally confused myself here. Nevermind that last commetn.